src/main/java/io/anserini/collection/C4Collection.java

```diff
@@ -171,5 +179,9 @@ public String raw() {
     public boolean indexable() {
       return true;
     }
+
+    public static void setId(String idS){
+      id = idS;
```

Comment:
common Java idiom is
`this.id = id` = okay to have the method arg have the same name.